7/debian9/7.6/addons-checksum

```diff
@@ -0,0 +1 @@
+82323cd7fbdd0e5d85cd71ceba7ca9ef1a5e7b56e5f037ecd22717b743ef9820  prometheus-exporter-1.0.0.jar
```

Comment:
Please remove this file, it's not needed

Reply:
done